docs/guides/becoming-a-cna-as-an-open-source-org-or-project.md

```diff
@@ -0,0 +1,186 @@
+
+Below are some of the benefits of becoming a CNA:
+
+* **Provide high-quality authoritative CVE Records for your users.** CVE Records created by third-parties can be incomplete or inaccurate. 
+* **CVEs can't be issued for projects in a CNA's scope without first reporting to the CNA.** This means that reporters _must_ initially engage with your CNA, thus reducing confusion and allowing subject-matter experts on the project and security policy to weigh in on whether to create a CVE for a given disclosure.
+* **Assign CVE IDs without needing to share embargoed information with other organizations.** This allows the project to determine for themselves  who, if anyone, needs or gets pre-disclosure information.
+
+## Requirements to become a CNA
+
+Before becoming a CNA you can look at the below set of requirements to make sure joining the CNA Program is feasible for your project or organization. **Remember that you can always leave and rejoin the CNA Program at a later date if circumstances change.**
+
+### Points of contact (POCs)
+
+Preferably 2 or more and ordered primary, secondary, etc. For each point of contact you’ll need the following information which is **not publishing publicly** and is only used internally by the CNA Program:
+
+* Name
+* Email address (Can’t be a group email address)
+* Phone number (**Required for primary POC only**, and only used for rare emergencies like [Known Exploited Vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) in scoped products)
+
+### Time and reporting obligations
+
+* **During CNA onboarding:**
+    * List of 3 available dates and times for a 1-hour call with CNA Program with all Points of Contact to answer questions and exercises. These three dates must be at least 3 weeks in the future.
+    * Complete practice exercises after the initial onboarding call and submit your exercises to the CNA Program for approval.
+
+* **Throughout CNA operations:**
+    * **No time requirements on timeline for [non-public aspects of coordinated vulnerability disclosure](https://www.cve.org/ResourcesSupport/AllResources/CNARules#appendix_d_disclosure_and_embargo_policies) (ie reporting, acknowledging, disclosures, etc).**
+    * CVE dispute requests have timeliness requirements:
+        * 3 days to acknowledge the dispute.
+        * 5 days after acknowledgement to make a decision or extend and inform the requester.
+        * 15 days after optional extension to make a final decision or escalate up to your Root CNA, resulting in a new dispute cycle.
+    * Get approval from Root CNA with changes to CNA record data like POCs, scope, vulnerability disclosure policy and location, etc.
+    * CNA inactivity policy: 6 months of no CVE IDs allocated will cause a few heartbeat emails sent to POCs. After three attempts to contact your organization will be removed from the CNA Program (and you can reapply to join again).
```

Comment:
Is this true? This year, 161 out of 320 CNAs have not published a CVE.  https://gist.github.com/jgamblin/42b7a3f2b18608835bb310b3b919f4cb

Reply:
My understanding is it's okay to not publish CVEs, responding to the heartbeats that indeed you are still active is all that's necessary. I am not certain however if heartbeats are being sent.

Reply:
I didn't get heartbeat as POC for Zowe organization despite us publishing the latest and allocating the latest ID more than 6 months ago now.

Reply:
Unless maybe test instance also counts.